adding_species_manuscript.tex

```diff
@@ -570,13 +568,13 @@ \subsection*{Basic setup for chromosome-level simulations}
 \subsection*{Extracting parameters from the literature}
 
 Simulations cannot of course precisely match reality, but in setting up simulations
-it is desireable to choose parameters that best reflect our current understanding.
+it is desireable to choose parameters that best reflect our current understanding of evolution.
```

Comment:
'understanding of evolution' is probably too big a goal here. Maybe replace with 'understanding of the evolutionary history of the species of interest'

Reply:
yes, because I think it is missing what we understand